### DISCUSSION TITLE
I hope I got this setup right :)

### DISCUSSION THREAD

**Discussion by NZ-Wanderer** [Rem0o/FanControl.Releases#619] [General]
My first time at trying this (got it after watching Jays video today)
<img width="944" alt="image" src="https://user-images.githubusercontent.com/447796/164891622-100f6056-163e-4c06-8963-18da1de9169c.png">

**Comment by Fr0stX76** [comment]
Looks good to me

Now tick"hide fan speed card" an untick "show hidden card" for a cleaner UI (from the top left menu, the 3 horizontal bars)

**Reply by Rem0o** [reply]
^ this

Also set your theme.

**Comment by NZ-Wanderer** [comment]
Thanks for the replies, much appreciated, I didn't know if I had the CPU one right as Jay did his with intel and his dropdown on what to choose for CPU was completely different to mine, so I just chose the embedded controller one hoping it was right..
And ummm I have chosen my theme, I ticked sync with system colors but nothing changed so just left it as it was..
Just changed it manually, and now it looks like this :)
<img width="747" alt="image" src="https://user-images.githubusercontent.com/447796/164946455-484b2163-21ff-4bd6-9e78-8113fd784e8d.png">

**Reply by Rem0o** [reply]
@Simpy The "sync" is only for dark theme/ light theme for now.
